src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheck.java

```diff
@@ -130,7 +130,7 @@ protected void processFiltered(File file, FileText fileText) {
         for (int i = 0; i < fileText.size(); i++) {
             final String line = fileText.get(i);
             final int realLength = CommonUtil.lengthExpandedTabs(
-                line, line.length(), getTabWidth());
+                line, line.codePointCount(0, line.length()), getTabWidth());
```

Comment:
I can only reproduce the issue with maven and not in Eclipse. Since I can't debug, I am starting to add more print outs to see the issue. I removed changes to CommonUtil and the issue remained.

Reply:
I opened the file in a hex editor and the line in question is 103 bytes (not include `\n`).

I believe the issue is the input file and that it may not be recognized in the right file format which you may be trying to use.

Reply:
@nmancus1 @romani The issue is the charset used to load the files. The default charset for Windows, since one isn't being supplied, is Cp1252 and thus is causing the files to be loaded using `windows-1252`.

To fix the issue, tests would need to specify a UTF-8 charset to Checker or you would need to use a code that is supported by both charsets and reproduces the reported issue.

Reply:
if we define this in test it means all Windows user who want to have this issue fixed, then they HAVE to define charset UTF-8 in config.
@rnveach , Am I get this right? 
if yes, we need to make ATTENTION note at issue.
and yes, define UTF8 in Test.

@nmancus1 , while we waiting for confirmation ... please add charset in Test with comment `// we need to set charset to let test pass on windows where default charset is not UTF-8`

Reply:
> @rnveach , Am I get this right?

Yes.

It will not affect just windows users, it is anyone who does not specify the right character set for their files. If you force linux chartset to be Cp1252, they would see the same issue too.

Reply:
I updated issue.